data/emission_factors.csv

```diff
@@ -1,100 +1,307 @@
+215,ZW,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.39607,
+216,global,public_transport,bus,mobitool,Stadtbus (9m),P.km,small,local,cng,kg/P.km,0.1464,
+217,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,medium,local,cng,kg/P.km,0.1034,
+218,global,public_transport,bus,mobitool,Stadtbus (18m),P.km,large,local,cng,kg/P.km,0.0709,
+219,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,average,local,cng,kg/P.km,0.1034,
+220,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,average,local,diesel,kg/P.km,0.1338,
+221,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,medium,local,diesel,kg/P.km,0.1338,
+222,global,public_transport,bus,mobitool,Stadtbus (18m),P.km,large,local,diesel,kg/P.km,0.0905,
+223,global,public_transport,bus,mobitool,Stadtbus (9m),P.km,small,local,diesel,kg/P.km,0.1716,
+224,global,public_transport,bus,mobitool,Reisebus,P.km,average,long-distance,diesel,kg/P.km,0.0465,
+225,global,public_transport,bus,mobitool,Trolleybus,P.km,average,local,electric,kg/P.km,0.0296,
+226,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,cng,kg/P.km,0.1528,
+227,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,cng,kg/P.km,0.1033,
+228,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,cng,kg/P.km,0.1233,
+229,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,average,kg/P.km,0.1864,
+230,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,gasoline,kg/P.km,0.186,
+231,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,gasoline,kg/P.km,0.1297,
+232,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,gasoline,kg/P.km,0.1506,
+233,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,gasoline,kg/P.km,0.2004,
+234,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,diesel,kg/P.km,0.189,
+235,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,diesel,kg/P.km,0.1109,
+236,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,diesel,kg/P.km,0.1322,
+237,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,diesel,kg/P.km,0.1757,
+238,global,public_transport,train,mobitool,Eisenbahn,P.km,,long-distance,,kg/P.km,0.0068,
+239,global,public_transport,train,mobitool,Eisenbahn,P.km,,local,,kg/P.km,0.008,
+240,global,public_transport,train,mobitool,Eisenbahn,P.km,,average,,kg/P.km,0.007,
+241,global,public_transport,train,mobitool,Strassenbahn,P.km,,local,electric,kg/P.km,0.0428,
+242,DE,public_transport,train,mobitool,Bahn Deutschland,P.km,,average,,kg/P.km,0.0408,
+243,FR,public_transport,train,mobitool,Bahn Frankreich,P.km,,average,,kg/P.km,0.0125,
+244,IT,public_transport,train,mobitool,Bahn Italien,P.km,,average,,kg/P.km,0.0747,
+245,AT,public_transport,train,mobitool,Bahn Österreich,P.km,,average,,kg/P.km,0.0151,
+246,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,hybrid,kg/P.km,0.1785,

```

Comment:
We will have to check if the "vehicle" vs. "public_transport" categories cause some issues with the current code and whether to change that.
In case we need to: Do you see any problems putting "transport" again for both "vehicle"s and "public_transport"?

Reply:
changed everything to transport